test/fit_tests/deploy/rackhd_stack_init.py

```diff
@@ -0,0 +1,343 @@
+'''
+Copyright 2015, EMC, Inc.
+
+Author(s):
+George Paulos
+
+This script initializes RackHD stack after install.
+'''
+
+
+import os
+import sys
+import subprocess
+# set path to common libraries
+sys.path.append(subprocess.check_output("git rev-parse --show-toplevel", shell=True).rstrip("\n") + "/test/fit_tests/common")
+import fit_common
+import pdu_lib
+
+# Locals
+MAX_CYCLES = 60
+
+class rackhd_stack_init(fit_common.unittest.TestCase):
+    def test01_preload_sku_packs(self):
+        print "**** Processing SKU Packs"
+        # Load SKU packs from GutHub
+        subprocess.call("rm -rf temp.sku; rm -rf on-skupack", shell=True)
+        os.mkdir("on-skupack")
+        # download all SKU repos and merge into on-skupack
+        for url in fit_common.GLOBAL_CONFIG['repos']['skupack']:
+            print "**** Cloning SKU Packs from " + url
+            subprocess.call("git clone " + url + " temp.sku", shell=True)
+            subprocess.call('cp -R temp.sku/* on-skupack; rm -rf temp.sku', shell=True)
+        # build build SKU packs
+        for subdir, dirs, files in os.walk('on-skupack'):
+            for skus in dirs:
+                if skus not in ["debianstatic", ".git"] and os.path.isfile('on-skupack/' + skus + '/config.json'):
+                    subprocess.call("cd on-skupack;mkdir -p " + skus + "/tasks " + skus + "/static "
+                                    + skus + "/workflows " + skus + "/templates", shell=True)
+                    subprocess.call("cd on-skupack; ./build-package.bash "
+                                    + skus + " " + skus + " >/dev/null 2>&1", shell=True)
+            break
+        # upload SKU packs to ORA
+        print "**** Loading SKU Packs to server"
+        for subdir, dirs, files in os.walk('on-skupack/tarballs'):
+            for skupacks in files:
+                print "\n**** Loading SKU Pack for " + skupacks
+                fit_common.rackhdapi("/api/1.1/skus/pack", action="binary-post",
+                                     payload=file(fit_common.TEST_PATH + "on-skupack/tarballs/" + skupacks).read())
+            break
+        print "\n"
+        # check SKU directory against source files
+        errorcount = 0
+        skulist = fit_common.json.dumps(fit_common.rackhdapi("/api/2.0/skus")['json'])
```

Comment:
This test use both API 2.0 (line 53) and API 1.1 (line 47), I think we'd use the same API version here. 2.0 is recommend since 1.1 will be deprecated.